CONTRIBUTING.md

```diff
@@ -80,4 +82,4 @@ if it is rendered as expected.
 
 Note that docstrings (documentation comments in the source code) are collected from the installed
 Optuna module. If you modified docstrings, make sure to install the module (e.g.,
-using `pip install -e .`) before building the documentation.
\ No newline at end of file
+using `pip install -e .`) before building the documentation.
```

Comment:
What's the diff on this line?

Reply:
I think that my editor automatically added a newline character to the end of this line.
I can remove this diff if you're concerned.

Reply:
Ah no worries, I was just curious since it wasn't rendered 🙂